AppCenter/AppCenter/Support/AppCenter.xcconfig

```diff
@@ -1,5 +1,5 @@
 PRODUCT_BUNDLE_IDENTIFIER = com.microsoft.appcenter
 
-OTHER_LDFLAGS = $(inherited) -framework Foundation -framework SystemConfiguration -lsqlite3
+OTHER_LDFLAGS = $(inherited) -framework Foundation -framework SystemConfiguration -lsqlite3 -fprofile-instr-generate
```

Comment:
Let's remove this `-fprofile-instr-generate` because "though may slow down your binary or make your binary invalid for the app store.". Definitely something we don't want.

Reply:
Please remove everywhere.